typescript/cli/src/rebalancer/README.md

```diff
@@ -0,0 +1,151 @@
+  # Used to prevent triggering new rebalances while a transfer is in progress
+  bridgeLockTime: 300000 # 5 minutes in ms
+
+  # Optional: Minimum amount to bridge (in wei)
+  # Used to prevent transferring small amounts that are not worth the gas cost
+  bridgeMinAcceptedAmount: 1000000 # 1 USDC (6 decimals)
+
+  # Optional: Set to true if the bridge is another Warp Route
+  # This is because bridges composed of other warp routes are interacted with differently
+  bridgeIsWarp: false
+
+  # Optional: Specify override configurations for specific chains
+  # This allows you to customize how this chain interacts with other particular chains
+  override:
+    arbitrumsepolia: # Chain name to override settings for
+      bridge: '0x4321...' # Use a different bridge when sending to this chain
+      bridgeLockTime: 300000 # 5 minutes in ms
+      bridgeMinAcceptedAmount: 1000000 # 1 USDC (6 decimals)
+      bridgeIsWarp: true
+
+  # Strategy-specific parameters (depending on rebalanceStrategy)
+  # Use one set of values for the strategy you are using
+
+  # For weighted strategy:
+  # Required: Relative weight for this chain
+  weight: 100 # (e.g All chains have equal weight, rebalancing will balance all chains to have the same amounts of collateral)
+  # Required: Determines how much deviation from the target amount is allowed before a rebalance is triggered (in percentage 0-100)
+  tolerance: 5 # 5% allows a 5% deviation from the target amount before a rebalance is needed
+
+  # For minAmount strategy (absolute):
+  # Absolute requires exact token amounts
+  # Required: Minimum amount to maintain on this chain (in wei)
+  minAmount: 100000000 # 100 USDC (6 decimals)
+  # Required: The objective value to rebalance to.
+  target: 110000000 # It should be bigger than `minAmount` to prevent immediate rebalance (110 USDC in this case)
+
+  # For minAmount strategy (relative):
+  # Relative requires percentage values. 0 = 0%, 0.5 = 50%, 1 = 100%.
+  # 100% represent a the sum of collaterals of rebalanceable amounts.
+  # Required: Minimum percentage to maintain on this chain.
+  minAmount: 0.3 # 30%
+  # Required: The objective value to rebalance to.
+  target: 0.35 # It should be bigger than `minAmount` to prevent immediate rebalance (35% in this case)
+```
+
+## Basic Usage
+
+To run the rebalancer, you need to provide:
+
+1. A configuration file
+2. A private key to sign rebalance transactions
+
+```bash
+# Using environment variable for private key
+HYP_KEY=your_private_key hyperlane warp rebalancer --configFile ./rebalancer-config.yaml
+
+# Using CLI option for private key
+hyperlane warp rebalancer --configFile ./rebalancer-config.yaml --key your_private_key
+```
+
+> **IMPORTANT**: A private key is REQUIRED for the rebalancer to function correctly. You can provide it either via the `--key` parameter or the `HYP_KEY` environment variable.
+
+### Overriding Configuration Options
```

Comment:
if there is no technical reason to keep overrides, please lean towards configs. In the CLI, we prefer to 1) use a config, which you have, and 2) reduce params